docs/protocol/js-ceramic/guides/ceramic-nodes/running-cloud.md

```diff
@@ -1,228 +1,267 @@
 
 ---
 
-To run your application on `mainnet` you'll need to run your own production-ready node or to use a community provider like [hirenodes](https://hirenodes.io/).
+To run your application on `mainnet` you'll need to run your own production-ready node or to use a community hosted nodes provider like [hirenodes](https://hirenodes.io/).
 
 ## Things to know
 
 ---
 
-**Ceramic networks** – There are currently three Ceramic networks: `mainnet`, `testnet-clay`, and `dev-unstable`. Learn more about each network [here](../../networking/networks.md). By default, Ceramic will connect to `testnet-clay` and a [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service) running on Gnosis. When you are ready to get on Ceramic `mainnet`, check out [this guide](../../../../composedb/guides/composedb-server/running-in-the-cloud) to get access to our `mainnet` anchor service.
+**Ceramic networks** 
+There are currently three main Ceramic networks: 
+- `mainnet`
+- `testnet-clay`
+- `dev-unstable`
 
-**Running IPFS** – Ceramic relies on a system called [IPFS](https://docs.ipfs.io/) to connect to and share data in Ceramic networks. IPFS runs as a separate process from the Ceramic node itself, with each Ceramic node connected to a dedicated IPFS node over HTTP. The Ceramic Daemon can launch an IPFS process automatically (referred to as running IPFS in "bundled" mode in the Ceramic config file), which is designed for testing and local development only. For production deployments you should run your own IPFS process manually and point your Ceramic node at it (referred to as running ipfs in "remote" mode in the Ceramic config file). This allows for more configuration options for your IPFS node allowing for more controlled resource allocation, as well as improved maintenance, debugging and observability. Note that Ceramic only supports `go-ipfs` version 0.12 or later.
+Learn more about each network [here](../../networking/networks.md). 
 
-The rest of this guide assumes you are running Ceramic with IPFS in "remote" mode.
+By default, Ceramic will connect to `testnet-clay` and a [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service) running on Gnosis. When you are ready to get on Ceramic `mainnet`, check out [this guide](../../../../composedb/guides/composedb-server/access-mainnet) to get access to our `mainnet` anchor service running on Ethereum mainnet.
 
-**Process management, restarts and data persistence** – Ceramic and IPFS will not automatically restart if they crash. You should configure your own restart mechanism, and you must ensure data persistence between restarts.
+**Supported platforms** – You can run Ceramic nodes on a cloud provider of your choice. This guide will include instructions for the Digital Ocean Kubernetes, but the
+instructions can be applied to the vast majority of other cloud providers like AWS and others.
+
+**Supported Operating Systems:**
+
+- Linux
+- Mac
```

Comment:
We do not support Window or Mac in the cloud. We only have Linux based docker images. Having details about Windows or Mac here is confusing since these instructions do not work with those platforms.